2996_reflection/reflection.md

```diff
@@ -2358,9 +2385,14 @@ namespace std::meta {
   consteval auto data_member_spec(info type_class,
                                   data_member_options_t options = {}) -> info;
   template <reflection_range R = initializer_list<info>>
-  consteval auto define_class(info type_class, R&&) -> info;
+    consteval auto define_class(info type_class, R&&) -> info;
+
+  // @[define_static_array](#define_static_array)@
+  template <typename T, input_range<T> R = initializer_list<T>>
+    consteval auto define_static_array(R&& args) -> span<>;
```

Comment:
This declaration is broken, what's the intent here?

Also, there's no `#define_static_array` section, no corresponding description of this function, and no wording. Split it up into its own PR?

Reply:
Oh man, that's super broken lol. Sure, we can do a separate PR.